package com.ExceptionHandling;

public class ExceptionH3  { 
	public static void main(String[] args) throws InterruptedException {

		int i[]  = new int[4];
		i[5] = 300;
		System.out.println("Caught exception");

	}
	
}
